lib/classes/PluginManager.js

```diff
@@ -99,9 +99,15 @@ class PluginManager {
 
   loadPlugins(plugins) {
     plugins.forEach((plugin) => {
-      const Plugin = require(plugin); // eslint-disable-line global-require
+      try {
+        const Plugin = require(plugin); // eslint-disable-line global-require
 
-      this.addPlugin(Plugin);
+        this.addPlugin(Plugin);
+      } catch (error) {
```

Comment:
Yep, I totally agree on that, I've already mentioned that here: https://github.com/serverless/serverless/issues/3956#issuecomment-316593002

I'll submit a fix asap.